protos/winch/winch.proto

```diff
@@ -0,0 +1,210 @@
+    rpc RelativeLengthControl(RelativeLengthControlRequest) returns(RelativeLengthControlResponse) {}
+
+    /*
+     * Wind or unwind line at specified rate.
+     */
+    rpc RateControl(RateControlRequest) returns(RateControlResponse) {}
+    
+    /*
+     * Perform the locking sequence to relieve motor while in the fully retracted position.
+     */
+    rpc Lock(LockRequest) returns(LockResponse) {}
+
+    /*
+     * Sequence of drop, slow down, touch down, reel up, lock.
+     */
+    rpc Deliver(DeliverRequest) returns(DeliverResponse) {}
+
+    /*
+     * Engage motor and hold current position.
+     */
+    rpc Hold(HoldRequest) returns(HoldResponse) {}
+
+    /*
+     * Return the reel to the fully retracted position.
+     */
+    rpc Retract(RetractRequest) returns(RetractResponse) {}
+
+    /*
+     * Load the reel with line.
+     *
+     * The winch will calculate the total loaded length and stop when the tension exceeds a threshold.
+     */
+    rpc LoadLine(LoadLineRequest) returns(LoadLineResponse) {}
+
+    /*
+     * Spool out the entire length of the line.
+     */
+    rpc AbandonLine(AbandonLineRequest) returns(AbandonLineResponse) {}
+
+    /*
+     * Spools out just enough to present the hook to the user to load the payload.
+     */
+    rpc LoadPayload(LoadPayloadRequest) returns(LoadPayloadResponse) {}
+
+}
+
+message SubscribeStatusRequest {}
+message StatusResponse {
+    Status status = 1; // The next 'winch status' state
+}
+
+message StatusFlags {
+    bool healthy = 1;           // Winch is healthy
+    bool fully_retracted = 2;   // Winch line is fully retracted
+    bool moving = 3;            // Winch motor is moving
+    bool clutch_engaged = 4;    // Winch clutch is engaged allowing motor to move freely
+    bool locked = 5;            // Winch is locked by locking mechanism
+    bool dropping = 6;          // Winch is gravity dropping payload
+    bool arresting = 7;         // Winch is arresting payload descent
+    bool ground_sense = 8;      // Winch is using torque measurements to sense the ground
+    bool retracting = 9;        // Winch is returning to the fully retracted position
+    bool redeliver = 10;        // Winch is redelivering the payload. This is a failover state if the line tension goes above a threshold during RETRACTING.
+    bool abandon_line = 11;     // Winch is abandoning the line and possibly payload. Winch unspools the entire calculated line length. This is a failover state from REDELIVER if the number of attempts exceeds a threshold.
+    bool locking = 12;          // Winch is engaging the locking mechanism
+    bool load_line = 13;        // Winch is spooling on line
+    bool load_payload = 14;     // Winch is loading a payload
+}
+
+message Status {
+    uint64 time_usec = 1;
+    float line_length_m = 2;
+    float speed_m_s = 3;
+    float tension_kg = 4;
+    float voltage_v = 5;
+    float current_a = 6;
+    int32 temperature_c = 7;
+    StatusFlags status_flags = 8;
```

Comment:
We need those as well otherwise `protoc-gen-mavsdk` will generate weird emptry docstrings :see_no_evil: 

```suggestion
    uint64 time_usec = 1; // Time in usec
    float line_length_m = 2; // Length of the line in meters
    float speed_m_s = 3; // Speed in meters per second
    float tension_kg = 4; // Tension in kilograms
    float voltage_v = 5; // Voltage in volts
    float current_a = 6; // Current in amperes
    int32 temperature_c = 7; // Temperature in Celsius
    StatusFlags status_flags = 8; // Status flags
```

Reply:
Thanks!!